toolchain/sem_ir/facet_type_info.h

```diff
@@ -103,27 +103,76 @@ constexpr FacetTypeInfo::RewriteConstraint
 struct IdentifiedFacetType {
   using RequiredInterface = SpecificInterface;
 
-  IdentifiedFacetType() : interface_id(InterfaceId::None), num_to_impl(0) {}
+  IdentifiedFacetType()
+      : interface_id_(InterfaceId::None), num_interface_to_impl_(0) {}
 
+  auto required_interfaces() const
+      -> const llvm::SmallVector<RequiredInterface>& {
```

Comment:
I think we normally use a view type here?
```suggestion
      -> const llvm::ArrayRef<RequiredInterface> {
```